geoutils/georaster.py

```diff
@@ -124,7 +124,7 @@ class Raster:
     transform: Affine
     crs: CRS
     nodata: int | float | None
-    res: list[float]
+    res: tuple[float]
```

Comment:
This should be `tuple[float, float]` as there are always just two items. Great that this is changed!

`tuple[float]` indicates just one item in the tuple. For a tuple with an unspecified length, use an ellipsis: `tuple[float, ...]`

This is why mypy starts complaining: 
```
tests/test_georaster.py:431: error: Tuple index out of range  [misc]
```